libbeat/processors/actions/decode_json_fields.go

```diff
@@ -165,6 +165,10 @@ func DecodeJSON(text []byte, to *interface{}) error {
 		return err
 	}
 
+	if dec.More() {
+		return errors.New("Multiple json elements found")
```

Comment:
error strings should not be capitalized. https://github.com/golang/go/wiki/CodeReviewComments#error-strings